src/components/navbar/Navbar/ProfileModal.tsx

```diff
@@ -130,11 +133,13 @@ type ButtonData = {
 function AccountContent({
   address,
   setCurrentState,
+  avatar,
   notification,
 }: ContentProps) {
-  const senderColor = useRandomColor(address)
+  const { authenticatedUser, signer } = useMyAccount((state) => state)
+  const senderColor = useRandomColor(signer?.address)
```

Comment:
why use signer.address when you can use address from props?

Reply:
We should be using the address from props. I should have replaced it before